app/consumer-democracy/app.go

```diff
@@ -758,50 +759,50 @@ func (app *App) SimulationManager() *module.SimulationManager {
 	return app.sm
 }
 
-// DemocConsumerApp interface implementations for e2e tests
+// DemocConsumerApp interface implementations for integration tests
 
 // GetConsumerKeeper implements the ConsumerApp interface.
 func (app *App) GetConsumerKeeper() consumerkeeper.Keeper {
 	return app.ConsumerKeeper
 }
 
-// GetE2eBankKeeper implements the ConsumerApp interface.
-func (app *App) GetE2eBankKeeper() e2e.E2eBankKeeper {
+// GetIntgBankKeeper implements the ConsumerApp interface.
+func (app *App) GetIntgBankKeeper() intgutil.IntgBankKeeper {
```

Comment:
This is trivial but I'd probably just call this `TestBankKeeper` (same prefix for other test keepers). E2E tests are treating Gaia like a black box anyway.

Reply:
Good point